libpromises/evalfunction.c

```diff
@@ -5119,69 +5119,85 @@ static FnCallResult FnCallFold(EvalContext *ctx,
 }
 
 /*********************************************************************/
-/* This function has been removed from the function list for now     */
-/*********************************************************************/
-#ifdef SUPPORT_FNCALL_DATATYPE
+
 static FnCallResult FnCallDatatype(EvalContext *ctx, ARG_UNUSED const Policy *policy, const FnCall *fp, const Rlist *finalargs)
```

Comment:
```suggestion
static FnCallResult FnCallType(EvalContext *ctx, ARG_UNUSED const Policy *policy, const FnCall *fp, const Rlist *finalargs)
```

Reply:
@olehermanse the name `FnCallType` is already used. Please see FnCall.h:78.